### PR TITLE
TASK: Add translations for placeholders

### PR DESCRIPTION
**Pull request by RafaelKa**
This change adds translations for all editor placeholders to XLIFF files.

### CODE REVIEW

**Comment by RafaelKa** [comment]
oooo master and 1.0 so different -> i must create new PR instead of this one or is it possible to fix that in other way?

**Comment by aertmann** [comment]
@RafaelKa: you can keep the PR, but you need to cherry-pick your change on top of the 1.0 branch and push it to your `seo-translations` branch.. then it will update this PR and work as intended..

**Comment by aertmann** [comment]
code wise, there's almost no difference between the branches but it's upmerged from 1.0 to master which causes such issues..

File: Documentation/conf.py [hunk 1]
```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = 'dev-master'
```
Comment (kdambekalns) [inline Documentation/conf.py]: This looks wrong on the 1.0 branch… also next change.

File: composer.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "description": "SEO configuration and tools for TYPO3 Neos",
     "license": "GPL-3.0+",
     "require": {
-        "typo3/neos": "~2.0"
+        "typo3/neos": "*"
```
Comment (kdambekalns) [inline composer.json]: Looks wrong on 1.0 branch.

**Comment by kdambekalns** [comment]
As it seems the branch was changed using the GitHub UI, this doesn't really work… See the list of commits contained now…

**Pull request closed without merging**